cli/account_command.go

```diff
@@ -236,12 +237,14 @@ func (c *actCmd) infoAction(_ *kingpin.ParseContext) error {
 		fmt.Printf("Account Usage:\n\n")
 		fmt.Printf("    Storage: %s\n", humanize.IBytes(info.Store))
 		fmt.Printf("     Memory: %s\n", humanize.IBytes(info.Memory))
-		fmt.Printf("    Streams: %s\n", humanize.IBytes(uint64(info.Streams)))
-		fmt.Printf("  Consumers: %s\n", humanize.IBytes(uint64(info.Consumers)))
+		fmt.Printf("    Streams: %d\n", info.Streams)
+		fmt.Printf("  Consumers: %d\n", info.Consumers)
 		fmt.Println()
 
 		fmt.Printf("Account Limits:\n\n")
 
+		fmt.Printf("   Max Message Payload: %s (%d bytes) \n\n", humanize.IBytes(uint64(nc.MaxPayload())), nc.MaxPayload())
```

Comment:
I think drop the printing of exact bytes. We never do that anywhere else I think?

Reply:
Sure, I had added that while testing something and forgot about it.

Reply:
Actually I remember now, I added the exact bytes because that is what `nsc describe account` does.